### ISSUE TITLE
Prometheus service test fails

### ISSUE THREAD

**Issue by bjornfor** [NixOS/nixpkgs#21316]
## Issue description
```
$ cd nixos/tests
$ nix-build nixos/tests/prometheus.nix 
error: The option `services.prometheus.scrapeConfigs.[definition 1-entry 1].target_groups' defined in `<unknown-file>' does not exist.
(use ‘--show-trace’ to show detailed location information)
```

There must be something wrong with the "target_groups" option.

Background: I was trying to setup a Prometheus instance. I struggled a bit with the Nix options (just wanted to paste some Yaml from a tutorial...), then got the Nix syntax mostly working but struggled with the missing "target_groups" option. Then I found there was even a test for this service, and that test fails :-)

## Technical details

* System: NixOS 17.03pre
* Nix version: 1.11.4
* Nixpkgs version: 5ed1aee3af6ce80c63e534fc26d7ee611d4ca50d (master as of today)

**Comment by bjornfor** [comment]
Doh, apparently target_groups got renamed to static_configs in d4599165010f1541219cf32278e21a18ef8db8f7.

**Comment by bjornfor** [comment]
Fixed in 3fb785b0ad6902d765601e489e5f695411d2752f.
